nodejs 18 agent builds fail

**Describe the bug**
NodeJS 18 builds fail in PRs.


```
This system is not registered with an entitlement server. You can use subscription-manager to register.

43 files removed
Removing intermediate container 7c7b7a840858
 ---> 283f5b5baf19
Step 13/15 : RUN npm config set registry=$nexusUrl/repository/npmjs/ &&     npm config set always-auth=true &&     npm config set _auth=$(echo -n $nexusAuth | base64) &&     npm config set email=no-reply@opendevstack.org &&     npm config set ca=null &&     npm config set strict-ssl=false &&     yarn config set registry $nexusUrl/repository/npmjs/ -g &&     echo node version: $(node --version) &&     echo npm version: $(npm --version) &&     echo npx version: $(npx --version) &&     echo yarn version: $(yarn --version)
 ---> Running in 77a1b6eb7a63
npm ERR! `always-auth` is not a valid npm option

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/jenkins/.npm/_logs/2023-04-19T14_47_13_740Z-debug-0.log
The command '/bin/bash -o pipefail -c npm config set registry=$nexusUrl/repository/npmjs/ &&     npm config set always-auth=true &&     npm config set _auth=$(echo -n $nexusAuth | base64) &&     npm config set email=no-reply@opendevstack.org &&     npm config set ca=null &&     npm config set strict-ssl=false &&     yarn config set registry $nexusUrl/repository/npmjs/ -g &&     echo node version: $(node --version) &&     echo npm version: $(npm --version) &&     echo npx version: $(npx --version) &&     echo yarn version: $(yarn --version)' returned a non-zero code: 1

Error: Process completed with exit code 1.
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -6,6 +6,7 @@
 - Removal of Centos agents ([#1209](https://github.com/opendevstack/ods-core/issues/1209))
 - Fix oauth-proxy sidecar image ([#862](https://github.com/opendevstack/ods-quickstarters/issues/862))
 - Upgrade be-gateway-nginx from fedora-rpm to rocky and from version 1.19 to 1.21 ([#880](https://github.com/opendevstack/ods-quickstarters/issues/880))
+- Fix nodejs 18 build by removing option always-auth ([#905](https://github.com/opendevstack/ods-quickstarters/issues/905))
 
 ## [4.1] - 2022-11-17
 

--- a/common/jenkins-agents/nodejs18/docker/Dockerfile.ubi8
+++ b/common/jenkins-agents/nodejs18/docker/Dockerfile.ubi8
@@ -45,7 +45,6 @@ RUN yum repolist \
     && yum clean all -y
 
 RUN npm config set registry=$nexusUrl/repository/npmjs/ && \
-    npm config set always-auth=true && \
     npm config set _auth=$(echo -n $nexusAuth | base64) && \
     npm config set email=no-reply@opendevstack.org && \
     npm config set ca=null && \

